src/test/librbd/test_mock_fixture.h

```diff
@@ -25,7 +25,11 @@ ACTION_P(CopyInBufferlist, str) {
 }
 
 ACTION_P2(CompleteContext, r, wq) {
+#if defined(__clang__)
+  ContextWQ *context_wq = static_cast<ContextWQ *>(wq);
```

Comment:
although this is a little bit awkward, but this is the best we can have with minimal change, otherwise we need to change some of the callers of `CompleteContext` to cast the `NULL` parameter to the `ContextWQ *`.

@dillaman what do you think?